reference/recency-surveillance.v1.json

```diff
@@ -228,7 +244,8 @@
                         "type": "string"
                       },
                       "health_unit_name": {
-                        "type": "string"
+                        "type": "string",
+                        "default": "text"
```

Comment:
@ssmusoke so I had this as text because I wouldn't see the concept answers for it, thoughts?